ui/page/dcrdex/dcrdex_page.go

```diff
@@ -87,6 +97,12 @@ func (pg *DEXPage) HandleUserInteractions() {
 	if pg.CurrentPage() != nil {
 		pg.CurrentPage().HandleUserInteractions()
 	}
+	if pg.splashPageInfoButton.Button.Clicked() {
+		pg.showInfoModal()
+	}
+	if pg.navigateToSettingsBtn.Button.Clicked() {
+		pg.WL.AssetsManager.SetDexFirstVisit(true)
```

Comment:
2 things.
1) Don't save to db. Just toggle a variable on this page for now.
2) After toggling, reload the window to update the display.